model/events/LtiAgsListener.php

```diff
@@ -59,6 +60,7 @@ public function onDeliveryExecutionStart(DeliveryExecutionCreated $event): void
             $taskQueue = $this->getServiceLocator()->get(QueueDispatcherInterface::SERVICE_ID);
             $taskQueue->createTask(new SendAgsScoreTask(), [
                 'registrationId' => $user->getRegistrationId(),
+                'deliveryExecutionId' => $deliveryExcecution->getIdentifier(),
```

Comment:
maybe check if $deliveryExcecution is an object (using instanceof) before using mehtod. Better safe than sorry.

Reply:
or if you are 100% sure then dont :D